src/main/java/com/synerzip/expenseCalculation/service/CategoryService.java

```diff
@@ -0,0 +1,36 @@
+package com.synerzip.expenseCalculation.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.synerzip.expenseCalculation.model.Category;
+import com.synerzip.expenseCalculation.repository.CategoryRepository;
+import com.synerzip.expenseCalculation.repository.ExpenseRepository;
+import com.synerzip.expenseCalculation.repository.UserRepository;
+
+@Service
+public class CategoryService {
+	
+	@Autowired
+	UserRepository userRepository;
+
+	@Autowired
+	CategoryRepository categoryRepository;
+
+	@Autowired
+	ExpenseRepository expenseRepository;
+	
+	public Category findByCategoryName(String categoryName) {
+		
+		Category category = categoryRepository.findByCategoryName(categoryName);
```

Comment:
what happen if category is not found? 
do exception handling for that and better to create your own custom exception which has message and error code.
do for all.